unstructured/partition/email.py

```diff
@@ -416,7 +416,12 @@ def partition_email(
                     raise ValueError(
                         "Specify the attachment_partitioner kwarg to process attachments.",
                     )
-                attached_elements = attachment_partitioner(filename=attached_filename)
+                attached_elements = attachment_partitioner(
+                    filename=attached_filename,
+                    metadata_last_modified=metadata_last_modified,
+                    max_partition=max_partition,
+                    min_partition=min_partition,
```

Comment:
passing through min_partition/max_partition here is a good catch. should there be a test confirming this is working as expected?

Reply:
Makes sense. I'll create one